tools/crm_mon.c

```diff
@@ -775,14 +775,25 @@ cib_connect(gboolean full)
 
     if (cib->state != cib_connected_query && cib->state != cib_connected_command) {
         crm_trace("Connecting to the CIB");
+
+        /* Hack: the CIB signon will print the prompt for a password if needed,
+         * but to stderr. If we're in curses, show it on the screen instead.
+         *
+         * @TODO Add a password prompt (maybe including input) function to
+         *       pcmk__output_t and use it in libcib.
+         */
```

Comment:
That would be nice.  While trying to get rid of output_format in crm_mon as much as possible, this one always caused me problems.

Reply:
Yep, it would help abstract away the terminal garbage for plain text too

Reply:
(in cib_remote_signon(), but obviously only once everything's on the new output model)